alfresco-docker-tika/src/test/java/org/alfresco/transformer/TikaControllerTest.java

```diff
@@ -300,6 +300,9 @@ public void dotDotSourceFilenameTest() throws Exception
         super.dotDotSourceFilenameTest();
     }
 
+
+//  Ignore test as source/target extension si mandatory parameter for Tika.
```

Comment:
Are there any planned changes that I might not be aware of or can this test be deleted instead of ignored if it's not needed anymore?

Reply:
Thank you - will be removed.